source/_integrations/otbr.markdown

```diff
@@ -12,6 +12,17 @@ ha_config_flow: true
 ---
 
 The Open Thread Border Router integration allows calling an Open Thread Border Router's REST API from Python and via WebSocket.
-The integration is automatically set up when the "Silicon Labs Multiprotocol" add-on is installed.
+
+This integration is installed automatically when the **Open Thread Border Router** add-on is installed. This happens when you do one of the following procedures:
```

Comment:
The add-on can also be used in combination with other hardware. I'd adjust it to something like this.

```suggestion
This integration is installed automatically when the **Open Thread Border Router** add-on is installed. For  Home Assistant Yellow or SkyConnect refer to the following procedures:
```